proposals/dark_mode_extension_icons.md

```diff
@@ -6,11 +6,11 @@ Feature to enable developers to enhance extension icon visibility in dark mode.
 
 **Document Metadata**
 
-**Author:** &lt;solomonkinard>
+**Author:** solomonkinard
 
-**Sponsoring Browser:** Chromium.
+**Sponsoring Browser:** Chromium, Safari, Firefox
 
-**Contributors:** &lt;oliverdunk>
+**Contributors:** oliverdunk, xeenon, carlosjeurissen
```

Comment:
Suggest adding myself (hanguokai) as a contributor since I have been actively promoting and participating in this issue for years.

Reply:
Done.